.kube/app/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM php:8.1-fpm
 
 ENV PHP_VERSION 8.1
-ENV NODE_VERSION 18
+ENV NODE_VERSION 18.18.2
```

Comment:
Make the version more explicit so directory it's installed will be correctly added to $PATH.